src/ontology/genepio-edit.owl

```diff
@@ -12644,6 +12705,89 @@ AnnotationAssertion(rdfs:label obo:GENEPIO_0100634 "State or Province or Territo
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100926 "isolated from single source")
+SubClassOf(obo:GENEPIO_0100926 obo:GENEPIO_0100286)
+
+# Class: obo:GENEPIO_0100929 (PONAR grab sampler)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100929 "A specimen collection device that is used to collect sediment samples from hard bottoms."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100929 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100929 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100929 "2023-11-29T14:21:43Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100929 "PONAR grab sampler")
+SubClassOf(obo:GENEPIO_0100929 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100930 (soil sample probe)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100930 "A specimen collection device that is used to collect a soil sample from the ground."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100930 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100930 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100930 "2023-11-29T12:15:31Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100930 "soil sample probe")
+SubClassOf(obo:GENEPIO_0100930 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100932 (scooping)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100932 "A specimen collection process that is used to dig out and move an amount of sample using a scoop."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100932 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100932 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100932 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100932 "2023-11-29T11:21:54Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100932 "scooping")
+SubClassOf(obo:GENEPIO_0100932 obo:OBI_0600005)
+SubClassOf(obo:GENEPIO_0100932 obo:OBI_0600012)
+
+# Class: obo:GENEPIO_0100933 (sediment collection)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100933 "A specimen collection process that is used to collect sediment samples from the seabed with either a commercial diver or a grab device."@en)
```

Comment:
sediment is deposited in both sea beds, and freshwater lakes, deltas, bogs etc. Is this term is to be specific to seabeds - or saltwater areas - it should say so in label.  Also, no need to mention the how "commercial diver" (or government one?) or the tool - grab device (or some other kind?).

Reply:
I agree, thanks for four feedback. 
Also, I think I should change the label to 'sediment sampling'.